text_mining.py

```diff
@@ -0,0 +1,132 @@
+"""
+Software Desgin 2017
+Text Mining Project
+
+Takes each president's inaugural addresses and compares their sentiments.
+The data is then plotted. Can be used to gauge the previous president's term.
+
+@author: Benjamin Ziemann, github: zneb97
+"""
+import requests
+import os
+import sys
+import pickle
+from bs4 import BeautifulSoup
+from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
+import re
+import plotly
+
+#Get plotly api key
+keyFile = open('plotlyAPIKey.txt','r')
```

Comment:
It would be easier to store API keys in a python file. Save the keys as variables in a python file and import the file to reference them.